### ISSUE TITLE
improve citation visibility

### ISSUE THREAD

**Issue by only1chunts** [gigascience/gigadb-website#83]
## user Story

>As a website user
>I want to see all the information pertaining to a dataset
>So that I can use it to further my research or education

## Acceptance Tests

```
@ok @javascript @citation-box
	Scenario: To see the citation box and and the citation source
		Given I am not logged in to Gigadb web site
		And I am on "/dataset/101001"
		When I press "Cite Dataset"
		Then I should see "Text"
		And I should see "RIS"
		And I should see "BibTeX"

	@ok @javascript @citation-box
	Scenario: To show the citation text from CrossCite after clicking the Formatted Text button in the citation box
		Given I am not logged in to Gigadb web site
		And I am on "/dataset/101001"
		And I press "Cite Dataset"
		When I click "Text"
		Then I go to new tab and should see "Huang, Y., Li, Y., Burt, D. W., Chen, H., Zhang, Y., Qian, W., Kim, H., Gan, S., Zhao, Y., Li, J., Yi, K., Feng, H., Zhu, P., Li, B., Liu, Q., Fairley, S., Magor, K. E., Du, Z., Hu, X., … Li, N. (2013). Genome sequence of the duck (Anas platyrhynchos) [Data set]. GigaScience Database. https://doi.org/10.5524/101001"

	@ok @javascript @citation-box
	Scenario: To get RIS file after clicking the RIS in the citation box
		Given I am not logged in to Gigadb web site
		And I am on "/dataset/101001"
		And I press "Cite Dataset"
		When I click on the "RIS" button
		Then the response should contain "101001"
		# 200: the request completed successfully
		And the response should contain "200"

	@ok @javascript @citation-box
	Scenario: To get BibTeX file after clicking the BibTeX in the citation box
		Given I am not logged in to Gigadb web site
		And I am on "/dataset/101001"
		And I press "Cite Dataset"
		When I click on the "BibTeX" button
		Then the response should contain "101001"
		# 200: the request completed successfully
		And the response should contain "200"
```

## Additional Info

We should improve the ease of citation by adding a "Cite Data" button like they do here: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/27672 in addition we should add the learn more about data citation link (http://best-practices.dataverse.org/data-citation/) to our FAQs page 

## Product Backlog Item Ready Checklist

* [ ] Business value is clearly articulated
* [ ] Item is understood enough by the IT team so it can make an informed decision as to whether it can complete this item
* [ ] Dependencies are identified and no external dependencies would block this item from being completed
* [ ] At the time of the scheduled sprint, the IT team has the appropriate composition to complete this item
* [ ] This item is estimated and small enough to comfortably be completed in one sprint
* [ ] Acceptance criteria are clear and testable
* [ ] Performance criteria, if any, are defined and testable
* [ ] The Scrum team understands how to demonstrate this item at the sprint review

## Product Backlog Item Done Checklist

* [ ] Code is complete
* [ ] Automated tests related to the changes are implemented and passing
* [ ] All automated test suites are passing locally
* [ ] Code is refactored to best practices and coding standards
* [ ] Documentation is updated as needed
* [ ] A Pull Request has been created and review requested
* [ ] Pull Request is reviewed and approved
* [ ] The item has been merged to the develop branch
* [ ] All automated test suites are passing on continuous Integration pipeline and item is ready to release

**Comment by only1chunts** [comment]
from @ScottBGI in a different issue ticket:
Can we also include "Cite This" functionality/tab in a similar manner to F1000Research use for their dataset box? These boxes look messy, but the cite button is useful and has a pop-up with citation details.

<img width="550" alt="screen shot 2018-09-05 at 15 09 52" src="https://user-images.githubusercontent.com/841708/45077200-22c15c00-b11f-11e8-825f-e392a3c481c4.png">

**Comment by rija** [comment]
closing as done and merged with PR #521
